tests/testdata/python-3.10.2/.python-version

```diff
@@ -0,0 +1 @@
+3.10.2
```

Comment:
This test case will fail until we have a pulumi release with https://github.com/pulumi/pulumi/pull/16855